### PR TITLE
Grammar and structure suggestions

### PR DESCRIPTION
**Pull request by creationlayer**

### CODE REVIEW

**Comment by btcdrak** [comment]
Please see my comments in #61

**Comment by creationlayer** [comment]
makes sense i'll update that. thanks

File: _posts/en/posts/2015-12-23-capacity-increases-faq.md [hunk 1]
```diff
@@ -192,7 +192,7 @@ For example,
   network to function well with fewer full nodes than it can under
   currently-deployed technology.
 
-The actual effect of these technologies is unknown, but scaling now with
+The actual affect of these technologies is unknown, but scaling now with
```
Comment (btcdrak) [inline _posts/en/posts/2015-12-23-capacity-increases-faq.md]: The original is correct /affect/effect/

File: _posts/en/posts/2015-12-23-capacity-increases-faq.md [hunk 2]
```diff
@@ -302,9 +302,9 @@ either. For example,
 
 The [BIP66][] (strict DER) soft fork which activated in July 2015 will
 soon be providing reduced processing time by making it possible to
-switch to libsecp256k1 for validation as described elsewhere is this
+switch to libsecp256k1 for validation as described elsewhere in this
 FAQ. The reduced validation time makes it uncommon among soft
-forks in providing direct benefits to miners.
+forks that it provides direct benefits to miners.
```
Comment (btcdrak) [inline _posts/en/posts/2015-12-23-capacity-increases-faq.md]: > forks in providing direct benefits to miners

maybe

forks in **that it** provid**es** direct benefits to miners

File: _posts/en/pages/2016-01-01-contribute.md [hunk 3]
```diff
@@ -23,7 +23,7 @@ please check the [contribution guidelines][CONTRIBUTING] to understand our workf
 
 **Discussion**
 
-Most Bitcoin Core related discussion happens in the IRC channels on irc.freenode.net `#bitcoin-core-dev`. There is also a mailing list for Bitcoin protocol discussion [bitcoin-dev][bitcoin-dev] and a general Bitcoin discussion [bitcoin-discuss][bitcoin-discuss]. We also have [Slack][slack] (request an [invitation][invite]).
+Most Bitcoin Core related discussion happens in IRC channels on irc.freenode.net `#bitcoin-core-dev`. There is also a mailing list for Bitcoin protocol discussion [bitcoin-dev][bitcoin-dev] and a general Bitcoin discussion [bitcoin-discuss][bitcoin-discuss]. We also have [Slack][slack] (request an [invitation][invite]).
```
Comment (btcdrak) [inline _posts/en/pages/2016-01-01-contribute.md]: Talking about one specific channel here /channels/channel/

> in IRC channels on irc.freenode.net `#bitcoin-core-dev`

in the `#bitcoin-core-dev` IRC channel on irc.freenode.net

File: _posts/en/posts/2015-12-23-capacity-increases-faq.md (hunk 2 above)
Comment (creationlayer) [inline _posts/en/posts/2015-12-23-capacity-increases-faq.md]: i agree

File: _posts/en/pages/2016-01-01-contribute.md (hunk 3 above)
Comment (creationlayer) [inline _posts/en/pages/2016-01-01-contribute.md]: the original copy said in the IRC channels, but specifies one seems vague. Are you pointing them to freenode in general, but specifying one, or meaning to list a couple.

File: _posts/en/pages/2016-01-01-contribute.md [hunk 4]
```diff
@@ -23,7 +23,7 @@ please check the [contribution guidelines][CONTRIBUTING] to understand our workf
 
 **Discussion**
 
-Most Bitcoin Core related discussion happens in the IRC channels on irc.freenode.net `#bitcoin-core-dev`. There is also a mailing list for Bitcoin protocol discussion [bitcoin-dev][bitcoin-dev] and a general Bitcoin discussion [bitcoin-discuss][bitcoin-discuss]. We also have [Slack][slack] (request an [invitation][invite]).
+Most Bitcoin Core related discussion happen in the `#bitcoin-core-dev` IRC channel on irc.freenode.net. There is also a mailing list for Bitcoin protocol discussion [bitcoin-dev][bitcoin-dev] and a general Bitcoin discussion [bitcoin-discuss][bitcoin-discuss]. We also have [Slack][slack] (request an [invitation][invite]).
```
Comment (btcdrak) [inline _posts/en/pages/2016-01-01-contribute.md]: if you change /happens/happen/ then you need to /discussion/discussions/, prefer not to change happens just correct the mistake that there are "channels".

**Comment by btcdrak** [comment]
ACK
thanks.

**Pull request merged**